app/_src/production/install-kumactl.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Install kumactl
+content_type: how-to
+---
+
+This how-to guide explains how to install `kumactl` in your environment.
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can do the following:
+
+* Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. 
+* Read and create resources in {{site.mesh_product_name}} in Universal mode.
+
+The `kumactl` binary is a client to the {{site.mesh_product_name}} HTTP API. 
+
+{% tabs install-kumactl useUrlFragment=false %}
+{% tab install-kumactl Kubernetes %}
+1. Download {{site.mesh_product_name}}:
+{% tabs install_kumactl useUrlFragment=false %}
+{% tab install_kumactl Script %}
+
+Run the following script to automatically detect the operating system and download Kuma:
+
+<div class="language-sh">
+  <pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -</code></pre>
+</div>
+
+You can omit the `VERSION` variable to install the latest version.
+{% endtab %}
+{% tab install_kumactl Direct Link %}
+
+Download the distribution manually. Download a distribution for the **client host** from where you will be executing the commands to access Kubernetes:
+
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-centos-amd64.tar.gz">CentOS</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-rhel-amd64.tar.gz">RedHat</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-debian-amd64.tar.gz">Debian</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-ubuntu-amd64.tar.gz">Ubuntu</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-darwin-amd64.tar.gz">macOS</a> or run `brew install kumactl`
+
+and extract the archive with `tar xvzf kuma-{{ page.latest_version }}.tar.gz`
+{% endtab %}
+{% endtabs %}
+
+1. Add the `kumactl` executable to your path:
```

Comment:
List items numbering is wrong, it should be 2 right?

![image](https://user-images.githubusercontent.com/2753650/223683455-89174fbb-47b3-4f8d-aeae-4c4e60b2c01c.png)